components/automate-cli/cmd/chef-automate/certRotate.go

```diff
@@ -182,7 +187,11 @@ func init() {
 
 func certRotateCmdFunc(flagsObj *certRotateFlags) func(cmd *cobra.Command, args []string) error {
 	return func(cmd *cobra.Command, args []string) error {
-		c := certRotateFlow{FileUtils: &fileutils.FileSystemUtils{}}
+		log, err := logger.NewLogger("text", "info")
+		if err != nil {
+			return err
+		}
+		c := certRotateFlow{FileUtils: &fileutils.FileSystemUtils{}, SSHUtil: sshutils.NewSSHUtilWithCommandExecutor(sshutils.NewSshClient(), log, command.NewExecExecutor())}
```

Comment:
Instead of this way can we create a function that takes these input as parameter and initialise and return pointer to this certRotateFlow. Making it more readable. ?

Reply:
done